### DISCUSSION TITLE
Windows升级到v0.3.27后，无法移动窗口

### DISCUSSION THREAD

**Discussion by supwang** [nextai-translator/nextai-translator#1339] [Q&A]
Windows升级到v0.3.27后，无法移动窗口，只能调整窗口大小，位置动不了，这个什么情况？
谢谢。

**Comment by yetone** [comment]
![image](https://github.com/openai-translator/openai-translator/assets/1206493/f798f9ac-87e6-4b0f-9607-bef0bf86b9e6)

由于 Tauri 的 bug 导致的，请拖动这个区域进行窗口移动

https://github.com/tauri-apps/tauri/issues/9023

**Reply by supwang** [reply]
谢谢！
